config/configuration.json

```diff
@@ -25,14 +24,14 @@
       "defaultPriority": "LOW"
     }
   },
-  "AreasWithHighwayTagCheck":{
-    "tag.filters":"highway->!pedestrian&area->!yes",
-    "challenge":{
-      "description":"A Highway should never be tagged as an area, all these tasks involve highways that have been.",
-      "blurb":"Highways should not be tagged as Areas.",
-      "instruction":"Remove highway tag from feature with area tag.",
-      "difficulty":"EASY",
-      "tags":"highway,landuse"
+  "AreasWithHighwayTagCheck": {
```

Comment:
It looks like spotless might have gone a bit overboard on cleaning up the config. Some of the changes are nice, though not all, and almost all are outside the scope of this PR. I am not entirely sure how we want to handle this. Perhaps @danielduhh has some suggestions.

Reply:
@danielduhh @Bentleysb The formatting of the config file happened as part of IntelliJ reformat file operation. I have reverted these changes and the config file should now look like the original with  additional config for ConflictingCarAccessibilityCheck.